docs/src/whatsnew/latest.rst

```diff
@@ -47,6 +47,13 @@ This document explains the changes made to Iris for this release
    differs from 0 or 2. This enables the use of this method on mesh
    coordinates. (:issue:`4672`, :pull:`4870`)
 
+#. `@lbdreyer`_ fixed the cube print out such that scalar ancillary variables
+   are displayed in a dedicated section rather than being added to section for
+   vector ancillary variables. Further, ancillary variables and cell measures
```

Comment:
```suggestion
   are displayed in a dedicated section rather than being added to the vector
   ancillary variables section. Further, ancillary variables and cell measures
```